docs/reST/ref/event.rst

```diff
@@ -229,9 +229,12 @@ and the ``joy`` attribute was deprecated.
 
    | :sl:`wait for a single event from the queue`
    | :sg:`wait() -> EventType instance`
+   | :sg:`wait(timeout) -> EventType instance`
 
    Returns a single event from the queue. If the queue is empty this function
-   will wait until one is created. The event is removed from the queue once it
+   will wait until one is created. If a ``timeout`` argument is given, the
+   function will return an event of type ``pygame.NOEVENT`` if no events enter the
+   queue in ``timeout`` milliseconds. The event is removed from the queue once it 
    has been returned. While the program is waiting it will sleep in an idle
    state. This is important for programs that want to share the system with
    other applications.
```

Comment:
These docs are great, but they need to mention somewhere that the timeout parameter only exists in pygame 2, just so our users still using pygame 1 don't get confused.

Usually we are doing this with one of these:

```
.. versionadded:: 2.0.0 ``timeout`` parameter
```

Added under the main block of documentation for the function, but I'd also add something like:

```
From pygame 2.0.0, if a ``timeout`` argument is given, the 
function will return an event of type ``pygame.NOEVENT`` 
if no events enter the queue in ``timeout`` milliseconds.
```

Reply:
Thanks, thats a good catch.
I looked around and in other places, the docs were using ``.. versionchanged::`` for new arguments, like in pygame.image.save and with all the new keyword arguments in draw functions. So I changed the events doc ``.. versionadded::`` to ``.. versionchanged::`` to be consistent with those other ones.